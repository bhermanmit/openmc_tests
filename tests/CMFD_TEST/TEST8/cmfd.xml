<?xml version="1.0"?>
<cmfd>

  <mesh>
    <origin> -15 -1 -1 </origin>
    <dimension> 30 1 1 </dimension>
    <width> 1.0 2.0 2.0 </width>
    <albedo> 0.0 0.0 1.0 1.0 1.0 1.0 </albedo>
<!--    <map> 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 </map> -->

    <map> 1 1 1 1 1 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 1 1 1 1 1 </map>
  </mesh>

</cmfd>
