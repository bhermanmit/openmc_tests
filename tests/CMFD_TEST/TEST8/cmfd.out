MESH (1. 1, 1)  GROUP 1           0.330000
MESH (1. 1, 1)  GROUP 2           -0.330000
MESH (2. 1, 1)  GROUP 1           3.78000E-02
MESH (2. 1, 1)  GROUP 2           -3.78000E-02
MESH (3. 1, 1)  GROUP 1           -1.41000E-02
MESH (3. 1, 1)  GROUP 2           1.41000E-02
MESH (4. 1, 1)  GROUP 1           -1.78000E-02
MESH (4. 1, 1)  GROUP 2           1.78000E-02
MESH (5. 1, 1)  GROUP 1           0.331000
MESH (5. 1, 1)  GROUP 2           -0.331000
MESH (6. 1, 1)  GROUP 1           -0.205841
MESH (6. 1, 1)  GROUP 2           0.206892
MESH (7. 1, 1)  GROUP 1           1.34152E-02
MESH (7. 1, 1)  GROUP 2           -1.07821E-02
MESH (8. 1, 1)  GROUP 1           8.20367E-02
MESH (8. 1, 1)  GROUP 2           -7.96761E-02
MESH (9. 1, 1)  GROUP 1           7.80984E-02
MESH (9. 1, 1)  GROUP 2           -8.10658E-02
MESH (10. 1, 1)  GROUP 1          9.80517E-02
MESH (10. 1, 1)  GROUP 2          -0.103242
MESH (11. 1, 1)  GROUP 1          0.117276
MESH (11. 1, 1)  GROUP 2          -0.118109
MESH (12. 1, 1)  GROUP 1          0.107304
MESH (12. 1, 1)  GROUP 2          -0.107782
MESH (13. 1, 1)  GROUP 1          9.36387E-02
MESH (13. 1, 1)  GROUP 2          -9.48025E-02
MESH (14. 1, 1)  GROUP 1          0.121867
MESH (14. 1, 1)  GROUP 2          -0.113763
MESH (15. 1, 1)  GROUP 1          0.113136
MESH (15. 1, 1)  GROUP 2          -0.104913
MESH (16. 1, 1)  GROUP 1          0.108092
MESH (16. 1, 1)  GROUP 2          -0.109993
MESH (17. 1, 1)  GROUP 1          0.100977
MESH (17. 1, 1)  GROUP 2          -9.95027E-02
MESH (18. 1, 1)  GROUP 1          0.110652
MESH (18. 1, 1)  GROUP 2          -0.109388
MESH (19. 1, 1)  GROUP 1          0.114552
MESH (19. 1, 1)  GROUP 2          -0.128616
MESH (20. 1, 1)  GROUP 1          8.62488E-02
MESH (20. 1, 1)  GROUP 2          -8.65922E-02
MESH (21. 1, 1)  GROUP 1          0.112035
MESH (21. 1, 1)  GROUP 2          -0.115673
MESH (22. 1, 1)  GROUP 1          0.109807
MESH (22. 1, 1)  GROUP 2          -0.115359
MESH (23. 1, 1)  GROUP 1          8.32871E-02
MESH (23. 1, 1)  GROUP 2          -7.20512E-02
MESH (24. 1, 1)  GROUP 1          3.37226E-02
MESH (24. 1, 1)  GROUP 2          -3.48414E-02
MESH (25. 1, 1)  GROUP 1          -0.224813
MESH (25. 1, 1)  GROUP 2          0.225716
MESH (26. 1, 1)  GROUP 1          0.317200
MESH (26. 1, 1)  GROUP 2          -0.317200
MESH (27. 1, 1)  GROUP 1          1.32000E-02
MESH (27. 1, 1)  GROUP 2          -1.32000E-02
MESH (28. 1, 1)  GROUP 1          -2.62000E-02
MESH (28. 1, 1)  GROUP 2          2.62000E-02
MESH (29. 1, 1)  GROUP 1          3.51000E-02
MESH (29. 1, 1)  GROUP 2          -3.51000E-02
MESH (30. 1, 1)  GROUP 1          0.338600
MESH (30. 1, 1)  GROUP 2          -0.338600
